### PR TITLE
16 summoner page

### PR DESCRIPTION
**Pull request by dshcheung**

### CODE REVIEW

File: app/assets/stylesheets/application.css.scss [hunk 1]
```diff
@@ -15,8 +15,8 @@
  */
 
 @import "variables";
-@import "bootstrap-sprockets";
-@import "bootstrap";
+// @import "bootstrap-sprockets";
+// @import "bootstrap";
 .body {
```
Comment (h8rry) [inline app/assets/stylesheets/application.css.scss:20]: Shouldn't put css in `application.css.scss`

File: app/controllers/summoners_controller.rb [hunk 2]
```diff
@@ -6,16 +6,21 @@ def find
 
     @tries = 0
     begin
-      url = "https://#{params['region']}.api.pvp.net/api/lol/#{params['region']}/v1.4/summoner/by-name/#{params['summoner']}?api_key=#{ENV['API_KEY']}"
+      url = "https://#{params['region']}.api.pvp.net/api/lol/#{params['region']}/v1.4/summoner/by-name/#{params['summoner'].downcase}?api_key=#{ENV['API_KEY']}"
```
Comment (h8rry) [inline app/controllers/summoners_controller.rb:9]: maybe break it down to several variables...so hard to read

File: app/controllers/summoners_controller.rb [hunk 3]
```diff
@@ -6,16 +6,21 @@ def find
...
       response = JSON.parse(open(url).read)
 
-      render json: {success: true}
+      if response.empty?
+        render json: {success: true, summoner: nil}, status: 200
+      else
+        render json: {success: true, summoner: response["#{params['summoner'].downcase}"]}, status: 200
```
Comment (h8rry) [inline app/controllers/summoners_controller.rb:15]: no jbuilder needed?

Comment (dshcheung) [inline app/controllers/summoners_controller.rb:15]: nope

**Pull request merged**